setup.py

```diff
@@ -20,7 +20,7 @@ def scan(dir, files=[]):
 import fileinput
 import re
 
-v_readme_regex = r'\[\_*v([0-9]+\.[0-9]+\.[0-9a-z]+)\_*\]'
+v_readme_regex = r'\[\_*v([0-9]+\.[0-9]+\.[0-9a-z\+]+)\_*\]'
```

Comment:
You can safely move the minor version upwards. No need to add `+`.

Reply:
Changed to 2.0.2